entrypoint.sh

```diff
@@ -70,6 +71,12 @@ git config --global user.name "$USER_NAME"
 
 {
 	git clone --single-branch --depth 1 --branch "$TARGET_BRANCH" "$GIT_CMD_REPOSITORY" "$CLONE_DIR"
+} || {
+  if [ "$CREATE_TARGET_BRANCH_IF_NEEDED" = 'true' ] ; then
+    git clone --single-branch --depth 1 "$GIT_CMD_REPOSITORY" "$CLONE_DIR"
+  else
+    false
```

Comment:
If the `create-target-branch-if-needed` is not set to `'true'` (which will be the default) we let the `try` block fails.